src/include/sof/schedule/ll_schedule.h

```diff
@@ -33,6 +33,12 @@
 
 #define ll_sch_get_pdata(task) task->private
 
+#if __XCC__
+#define TICKS_NEEDED_TO_FINISH_RESCHEDULE 150
+#else
+#define TICKS_NEEDED_TO_FINISH_RESCHEDULE 300
```

Comment:
@mrajwa @tlauda imho it should be platform/arch value

Reply:
I think it will be overkill in this scenario - it's true that this value is architecture dependent (and we have it wrapped up by precompiler-architecture check_ but in the same time it is dependent on scheduler implementation.

Reply:
@mrajwa it's platform/arch level config, lets put in all in one global place.